src/pages/instances/actions/DeleteInstanceBtn.tsx

```diff
@@ -21,7 +21,13 @@ const DeleteInstanceBtn: FC<Props> = ({ instance }) => {
       .then(() => {
         navigate(
           `/ui/${instance.project}/instances`,
-          notify.queue(notify.success(`Instance ${instance.name} deleted.`))
+          notify.queue(
+            notify.success(
+              <>
+                Instance <ItemName item={instance} bold /> deleted.
```

Comment:
Ouch, right. This makes me unhappy though, because we can't respect the pattern that the name should be highlighted in bold. We should find a (general) solution for this, but probably out of the scope of this PR, so I will restore the string for the time being.